app/frontend/src/features/messages/groupchats/GroupChatsTab.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { Provider } from "react-redux";
+import { store } from "../../../store";
+import GroupChatsTab from "./GroupChatsTab";
+
+import MessageView, { MessageProps } from "../messagelist/Message";
+import { mockedService } from "../../../__mocks__/service";
+
+Object.assign(mockedService, {
+  conversations: {
+    listGroupChats: () =>
+      Promise.resolve([
+        {
+          groupChatId: 3,
+          title: "groupchattitle",
+          memberUserIdsList: [],
+          adminUserIdsList: [],
+          onlyAdminsInvite: true,
+          isDm: false,
+          // created?: google_protobuf_timestamp_pb.Timestamp.AsObject,
+          unseenMessageCount: 0,
+          lastSeenMessageId: 4,
+          latestMessage: message,
+        },
+      ]),
+    getGroupChatMessages: () => Promise.resolve([message, message2]),
+  },
+});
+
+export default {
+  title: "GroupChatsTab",
+  component: GroupChatsTab,
+  argTypes: {},
+  decorators: [
+    (storyFn) => {
+      return <Provider store={store}>{storyFn()}</Provider>;
+    },
+  ],
+} as Meta;
+
+const Template: Story<any> = (args) => <GroupChatsTab {...args} />;
```

Comment:
```suggestion
const Template: Story = (args) => <GroupChatsTab {...args} />;
```
Since this component doesn't actually take any props so it's fine to leave it at the Storybook default?